### PR TITLE
Add default layer OSM mapsurfer roads

### PR DESCRIPTION
**Pull request by sonlinux**
fix kartoza/LEDET_BIMS#65

The rest of the layers were added by @meomancer in https://github.com/kartoza/django-bims/pull/15
https://staging.limpopobims.kartoza.com/map
layers

I was adding the remaining layer 
- OSM mapsurfer roads

ezgif com-optimize

### CODE REVIEW

**Comment by dimasciput** [comment]
@sonlinux can you provide some screenshots?

**Comment by sonlinux** [comment]
Yes am just compressing the gif its bigger for upload

**Comment by sonlinux** [comment]
So far all the layers have been sourced from different sources.
It would be nicer if we had a consistent source for all the base layers.

File: bims/static/js/views/olmap_basemap.js [hunk 1]
```diff
@@ -100,11 +108,22 @@ define(['backbone', 'underscore', 'jquery', 'ol', 'olMapboxStyle'], function (Ba
                 baseSourceLayers.push(bingMap);
             }
 
+            // OSM MAPSURFER ROADS - Make default
+            var mapSurfer = new ol.layer.Tile({
+                title: 'OSM Mapsurfer roads',
+                source: new ol.source.XYZ({
+                    attributions: ['OpenStreetMap contributors, ODbL, Imagery GIScience Research Group @ Heidelberg University'],
+                    url: 'http://korona.geog.uni-heidelberg.de/tiles/roads/x={x}&y={y}&z={z}'
```
Comment (dimasciput) [inline bims/static/js/views/olmap_basemap.js]: can you find the https version?

Comment (sonlinux) [reply bims/static/js/views/olmap_basemap.js]: Alright
Ill take note of that lets-encrypt effect.
Thanks

**Comment by sonlinux** [comment]
Done @dimasciput

**Pull request merged**